### PR TITLE
Clean up lint ignore list

### PR DESCRIPTION
**Pull request by nikhilvenkatraman-okta**
Resolves: OKTA-399777

## Description:

- Clean up lint ignore list for already resolved tickets and added comments for existing ones 

## PR Checklist

- [x] Have you verified the basic functionality for this change?

### Reviewers:

@jmelberg-okta @TianGan-okta 

### Issue:

- [OKTA-399777](https://oktainc.atlassian.net/browse/OKTA-399777)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -99,13 +99,10 @@ module.exports = {
...
         {
           'files': [
-            'playground/**/authenticator-verification-okta-verify-reject-push.json',
-            'playground/**/error-with-failure-redirect.json',
-            'playground/**/error-authenticator-webauthn-failure.json',
-            'playground/**/error-internal-server-error.json',
-            'playground/**/error-okta-verify-totp.json',
-            'playground/**/error-user-is-not-assigned.json',
-            'playground/**/identify-unknown-user.json',
+            'playground/**/error-with-failure-redirect.json', //OKTA-390647
```
Comment (nikhilvenkatraman-okta) [inline .eslintrc.js:102]: authenticator-verification-okta-verify-reject-push.json: https://oktainc.atlassian.net/browse/OKTA-389438

error-authenticator-webauthn-failure.json:  https://github.com/okta/okta-signin-widget/commit/9e78e652f605255aa3447ae664e5f42611969963

error-okta-verify-totp.json: https://oktainc.atlassian.net/browse/OKTA-389463

File: packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js [hunk 2]
```diff
@@ -23,6 +23,8 @@ module.exports = {
       'idx.email.verification.required', // mapped to v1 key
       'security.access_denied', // mapped to v1 key in i18nTransformer
       'api.users.auth.error.POST_PASSWORD_UPDATE_AUTH_FAILURE', // mapped to oie alias in i18Transformer
+      'api.authn.poll.error.push_rejected',// mapped to oktaverify.rejected in i18Transformer
+      'authfactor.webauthn.error.assertion_validation_failure' // mapped to authfactor.webauthn.error in i18Transformer
```
Comment (jmelberg-okta) [inline packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js:27]: Did we look into updating our lint rule to read from the `i18nTransformer` object? Would cut down on the touch points for developers.

**Review by jmelberg-okta — APPROVED** [review]

**Review by TianGan-okta — APPROVED** [review]
LGTM

File: packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js (hunk 2 above)
Comment (nikhilvenkatraman-okta) [reply packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js:27]: Good point, not yet but I have  a ticket for it https://oktainc.atlassian.net/browse/OKTA-399792. Will follow up

**Pull request closed without merging**